lib/git-manager.js

```diff
@@ -147,6 +159,17 @@ module.exports = {
          number: pullNumber
       }).then(getAllPages);
    },
+
+   getPullReviewComments: function(pullNumber) {
+      // "review" comments are the comments on the diff of pull
+      var getPullComments = Promise.denodeify(github.pullRequests.getComments);
+      return getPullComments({
+         user: config.repo.owner,
+         repo: config.repo.name,
+         number: pullNumber
```

Comment:
This could prob be DRYd up (noticed we copy pasta the config in each of these functions) but for a future pull